ui/tests/acceptance/job-definition-test.js

```diff
@@ -108,7 +112,9 @@ module('Acceptance | job definition', function (hooks) {
   });
 
   test('when the job for the definition is not found, an error message is shown, but the URL persists', async function (assert) {
+    assert.expect(4);
     await Definition.visit({ id: 'not-a-real-job' });
+    await percySnapshot(assert);
```

Comment:
Note: I think this is the one taking the snapshot of the error page discussed earlier